src/ethereum_test_tools/common/types.py

```diff
@@ -455,10 +455,10 @@ def alloc_to_accounts(got_alloc: Dict[str, Any]) -> Mapping[str, Account]:
     accounts = {}
     for address, value in got_alloc.items():
         account = Account(
-            nonce=int(value.get("nonce", None), 16)
-            if "nonce" in value
+            nonce=int_or_none(value["nonce"]) if "nonce" in value else None,
```

Comment:
I think `int_or_none(value.get("nonce", None))` would work here.

Same for balance below.